build.gradle.kts

```diff
@@ -0,0 +1,59 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+plugins {
+	id("org.springframework.boot") version "2.2.5.RELEASE"
+	id("io.spring.dependency-management") version "1.0.9.RELEASE"
+	id("org.asciidoctor.convert") version "1.5.8"
+	kotlin("jvm") version "1.3.61"
+	kotlin("plugin.spring") version "1.3.61"
+}
+
+group = "com.wirvsvirus"
+version = "0.0.1-SNAPSHOT"
+java.sourceCompatibility = JavaVersion.VERSION_11
+
+val developmentOnly by configurations.creating
+configurations {
+	runtimeClasspath {
+		extendsFrom(developmentOnly)
+	}
+}
+
+repositories {
+	mavenCentral()
+}
+
+extra["snippetsDir"] = file("build/generated-snippets")
```

Comment:
Ich musste hier 
```
val snippetsDir = file("build/generated-snippets")
extra["snippetsDir"] = snippetsDir
```
draus machen, weil ich sonst unten in Zeile 53 und 57 Fehler hatte